src/authentication/user/middleware/index.ts

```diff
@@ -3,29 +3,27 @@ import logger from "../../../logger";
 import * as pageURLs from "../../../model/page.urls";
 
 export default (req: Request, res: Response, next: NextFunction) => {
+  const referringPageURL = req.header("Referer") as string;
 
-  logger.debug("Check if user is authenticated");
+  logger.debug("Check if user has referrer ");
+  if (referringPageURL === undefined) {
```

Comment:
This new logic slightly changes the behaviour  of this story (which is 'done') - https://companieshouse.atlassian.net/browse/LFA-1215. This new way of doing things may well be fine, but worth checking with analyst and tester.

Reply:
Have discussed with Tester and will also discuss with analyst before merging

Reply:
Have discussed with analyst and they are happy, merging now